client/packages.go

```diff
@@ -51,36 +51,36 @@ const (
 	TypeOS        = "os"
 )
 
-// Packages returns the list of packages the system can handle
-func (client *Client) Packages() (map[string]*Package, error) {
-	const errPrefix = "cannot list packages"
+// Snaps returns the list of snaps the system can handle
```

Comment:
Not really part of this pull-request, but I wonder what "list of snaps the system can handle" means? I mean, I guess the system can handle snaps that are not included here :)